src/phylo/node_basics.jl

```diff
@@ -0,0 +1,335 @@
+# phylo/node_basics.jl
+# ====================
+#
+# Types and methods for phylogenetic trees.
+#
+# Part of the Bio.Phylo module.
+#
+# This file contains the methods for updating or querying characteristics of
+# nodes/clades in a phylogeny. In the Phylo representation these are
+# vertices of a LightGraphs DiGraph, which are numbered higher than the
+# number of taxa + 1.
+#
+# This file is a part of BioJulia. License is MIT: https://github.com/BioJulia/Bio.jl/blob/master/LICENSE.md
+
+
+
+"""
+    is_root(tree::Phylogeny, vertex::Int)
+
+Test whether a vertex is the root.
+
+# Examples:
+```julia
+is_root(my_tree, 9)
+```
+"""
+function is_root(tree::Phylogeny, vertex::Int)
+    return vertex == root_vertex(tree)
+end
+
+
+"""
+    is_root{T <: SymString}(tree::Phylogeny, vertex::T)
+
+Test whether a vertex is the root.
+
+# Examples:
+```julia
+is_root(my_tree, :Root)
+is_root(my_tree, :A)
+```
+"""
+function is_root{T <: SymString}(tree::Phylogeny, vertex::T)
```

Comment:
No need to use the type variable. Just `is_root(tree::Phylogeny, vertex::SymString)` would be fine.